src/wujihandpy/__init__.py

```
@@ -3,4 +3,5 @@
 from ._core import *
 from ._version import __version__
 
-__all__ = ["__version__", 'Hand', 'Finger', 'Joint', 'IController', 'filter']
+__all__ = ["__version__", 'Hand', 'Finger', 'Joint', 'IController', 'filter',
+           'TactileBoard', 'TactileFrame', 'TactileHandedness']
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)